platforms/ide/problems/src/main/kotlin/org/gradle/problems/internal/impl/DefaultProblemsReportCreator.kt

```diff
@@ -74,15 +74,15 @@ class DefaultProblemsReportCreator(
                             property("totalProblemCount", problemCount.get())
                             buildNameProvider.buildName()?.let { property("buildName", it) }
                             property("requestedTasks", taskNames.joinToString(" "))
-                            property("documentationLink", DocumentationRegistry().getDocumentationFor("problem-report"))
+                            property("documentationLink", DocumentationRegistry().getDocumentationFor("problems-report"))
                             property("documentationLinkCaption", "Problem report")
                         }
                     }
                 }
             }
         })?.let {
             val url = ConsoleRenderer().asClickableFileUrl(it)
-            logger.warn("Problems report is available at: $url")
+            logger.warn("Problem report (Experimental) is available at: $url")
```

Comment:
🤔 

I'd suggest to add a prefix instead

    logger.warn("[Experimental] Problem report is available at: $url")

or

    logger.warn("[Incubating] Problem report is available at: $url")